Extensions/xkit_patches.js

```diff
@@ -1268,9 +1481,13 @@ XKit.tools.getParameterByName = function(name){
 
 			},
 
+			/**
+			 * Switches control button between "disabled" and normal.
```

Comment:
"switches" implies that this is a toggle, which it isn't—it depends on the passed value.